tests/tests_integration_pytest.yml

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Empty play to gather facts
+  hosts: all
```

Comment:
Is this really needed?

Reply:
This is needed when using `when: ... ansible_distribution` with `import_playbook`